apps/docs/src/app/platform/component-docs/platform-forms/platform-textarea/platform-textarea-examples/platform-textarea-counter-example.component.html

```diff
@@ -0,0 +1,77 @@
+<main class="fd-page">
+    <div *ngIf="true" class="fd-page__content fd-has-background-color-neutral-2">
+        <fdp-form-group #fg2 [multiLayout]="true" [formGroup]="form">
+            <fdp-form-field
+                #ff2
+                [id]="'detailedDescription'"
+                label="Detailed description"
+                [placeholder]="'Start entering detailed description'"
+                hint="This is tooltip help"
+                zone="zLeft"
+                rank="10"
+                hintPlacement="'left'"
+                [required]="true"
+                [validators]="textareaValidator"
+            >
+                <fdp-textarea
+                    [formControl]="ff2.formControl"
+                    [name]="'detailedDescription'"
+                    [growingMaxLines]="3"
+                    [growing]="true"
+                    [maxLength]="10"
+                    [cols]="10"
+                    [stateType]="'error'"
+                    [showExceededText]="true"
+                    [height]="'80px'"
+                    [wrapType]="'hard'"
+                    value="Lorem ipsum, dolor sit amet"
+                >
+                </fdp-textarea>
+            </fdp-form-field>
+
+            <fdp-form-field
+                #ff3
+                [id]="'noCounterMessageInteraction'"
+                label="When showExceededText is false and value is set"
+                [placeholder]="
+                    'Start entering. Counter message will not be shown, but if an initial value is set whose length > maxLength, excess text is removed on Backpress'
+                "
+                hint="This is tooltip help"
+                zone="zLeft"
+                rank="10"
+                hintPlacement="'left'"
+                [required]="true"
+                [validators]="textareaNoCounterMessageValidator"
+            >
+                <fdp-textarea
+                    [formControl]="ff3.formControl"
+                    [name]="'noCounterMessageInteraction'"
+                    [growingMaxLines]="3"
+                    [growing]="true"
+                    [maxLength]="10"
+                    [cols]="10"
+                    [showExceededText]="false"
+                    [wrapType]="'hard'"
+                    value="Some value that crosses max length"
+                >
+                </fdp-textarea>
+            </fdp-form-field>
+
+            <fdp-form-field [id]="'submit-button'" rank="20" zone="zLeft">
```

Comment:
If you are showing only growing option, do not include character count. It will confuse the user

Reply:
I will remove the growing options from this example. 
In the autogrow example, I have used a combination just to show the behaviour when both counter and growing options are enabled. Hope that is okay.